modules/client/src/client-common/context.ts

```diff
@@ -221,10 +250,10 @@ export class Context {
 
   private static resolveWeb3Providers(
     endpoints: string | JsonRpcProvider | (string | JsonRpcProvider)[],
-    network: Networkish,
+    network: Networkish
```

Comment:
These formatting glitches look like using a formatter other than `deno` :)

Reply:
I guess it would be good to define the formatter somewhere.... 
In most projects, we use prettier.
And when I run `demo fmt` there are a lot of files not formatted properly
![image](https://user-images.githubusercontent.com/4057473/200865956-971ecf77-fb9e-47f6-b215-dfd1200171a1.png)
and the linter fails with
```
✖ 569 problems (569 errors, 0 warnings)
  569 errors and 0 warnings potentially fixable with the `--fix` option.
```

I suggest we create another task to check which formatter, formatter config, linter and linter config we wanna use.

Reply:
No priority here, just trying to avoid the diff containing formatting changes only. For TS, deno was defined as the formatter in the coding guidelines. For the rest of files, either prettier or no formatting (markdown) is fine.